lib/order.rb

```diff
@@ -1,3 +1,9 @@
+require "csv"
+require "pry"
+
+# access CSV and other type use require relative
+FILE_NAME = "../support/orders.csv"
```

Comment:
1.  Good that you're using a constant!
1.  The path is off.  You should write it from the perspective of whomever is running the app.  Users should run `rake` from the project's home folder, so the path to the CSV from the home folder is `support/orders.csv`

So this should be:
```ruby
FILENAME = "support/orders.csv"
```